types/jquery.pnotify/index.d.ts

```diff
@@ -323,8 +323,22 @@ interface PNotify {
 
 interface PNotifyConstructor {
     new (options?: PNotifyOptions): PNotify;
-
+    
+    /**
+     * Remove all notices.
+     */
     removeAll(): void;
+
+    /**
+     * Remove all the notices in a stack.
+     * @param stack 
+     */
+    removeStack(stack: PNotifyStack): void;
+    
+    /**
+     * Reposition the notices, optionally animating their movement.
+     */
+    positionAll(animate: string): void
```

Comment:
the animate parameter of positionAll appears to be an optional boolean
You can find sample usages here: https://github.com/sciactive/pnotify/blob/ffb47b2b4f1ed4aaf6c60db0b84d6218a05dfd22/src/pnotify.js#L56
https://github.com/sciactive/pnotify/blob/ffb47b2b4f1ed4aaf6c60db0b84d6218a05dfd22/src/pnotify.js#L204

Reply:
good catch forgot to put it even the documentations say *optionally animating their movement*